tests/core/test_alloca.c

```diff
@@ -13,9 +13,9 @@ int main(int argc, char **argv) {
   char *pc, *pc2;
   assert(argc == 1);
   pc = (char *)alloca(4+argc);
-  assert(((int)pc) % 4 == 0);
+  assert(((long)pc) % 4 == 0);
```

Comment:
Is this case needed?  Can't it just be removed?

Reply:
Was just trying to fix the existing test for 64-bit. It probably shouldn't be removed since this also still runs for 32-bit.